Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -263,6 +263,10 @@ class RulesTests: XCTestCase {
         verifyRule(ReturnArrowWhitespaceRule.description)
     }
 
+    func testSortedImports() {
```

Comment:
you should add this to `allTests` in the end of this file to make it run on Linux as well